controllers/selfnoderemediationconfig_controller.go

```diff
@@ -38,6 +40,12 @@ import (
 	"github.com/medik8s/self-node-remediation/pkg/render"
 )
 
+const (
+	dsName                   = "self-node-remediation-ds"
+	lastChangedAnnotationKey = "dsLastChangeVersion"
+	lastChangeAnnotationVal  = "rhwa-23.3"
```

Comment:
typo: `lastChange` -> `lastChanged`

Reply:
why do you consider it a typo ?
Can't you say something like `what was the last change ? ` ?

Reply:
Yes, it was just that you have used `lastChanged` in few other places, so I though you prefer `lastChanged` instead of `lastChange` 🤷🏻‍♂️